spec/models/family_spec.rb

```diff
@@ -18,55 +18,29 @@
   it { should validate_presence_of(:donation_deadline) }
 
   it "returns correct number of people" do
-    nationality = Nationality.create(photo_path: "x",
-      info_link: "x",
-      greeting: "x",
-      name: "Syrian")
-    family = Family.create(first_name: "TestFirst",
-      last_name: "TestLast",
-      arrival_date: 10.days.from_now,
-      donation_deadline: 5.days.from_now,
-      nationality_id: nationality.id,
-      num_married_adults: 2,
-      num_unmarried_adults: 1,
-      num_children_over_two: 2,
-      num_children_under_two: 1)
+    family = create(:family,  num_married_adults: 2,
+                              num_unmarried_adults: 1,
+                              num_children_over_two: 2,
+                              num_children_under_two: 1)
 
     expect(family.num_people).to eq(6)
   end
 
   it "returns correct number of adults" do
-    nationality = Nationality.create(photo_path: "x",
-      info_link: "x",
-      greeting: "x",
-      name: "Syrian")
-    family = Family.create(first_name: "TestFirst",
-      last_name: "TestLast",
-      arrival_date: 10.days.from_now,
-      donation_deadline: 5.days.from_now,
-      nationality_id: nationality.id,
-      num_married_adults: 2,
-      num_unmarried_adults: 1,
-      num_children_over_two: 2,
-      num_children_under_two: 1)
+    family = create(:family,  num_married_adults: 2,
+                              num_unmarried_adults: 1,
+                              num_children_over_two: 2,
+                              num_children_under_two: 1)
+
```

Comment:
I'm glad you cleaned this up to use the factories. There are several places I think we should be able to do this.